mix/QContractDefinition.cpp

```diff
@@ -44,3 +44,14 @@ QContractDefinition::QContractDefinition(dev::solidity::ContractDefinition const
 		m_functions.append(new QFunctionDefinition(it->second, i));
 }
 
+
+QFunctionDefinition* QContractDefinition::getFunction(dev::FixedHash<4> _hash)
+{
+	if (m_constructor->hash() == _hash)
```

Comment:
Is that really used? Note that it is possible by inheritance that the final contract has a function that has the same hash as the constructor (tough, to be honest, I'm not sure whether that really works) and can be called after creation:
```
contract base { function f() { } }
contract f is base { function f() {} }
```